packages/flutter/test/material/text_form_field_test.dart

```diff
@@ -1169,4 +1169,32 @@ void main() {
     final EditableText editableText = tester.widget(find.byType(EditableText));
     expect(editableText.magnifierConfiguration, equals(myTextMagnifierConfiguration));
   });
+
+
+    testWidgets('Error color for cursor while validation', (WidgetTester tester) async {
+
+    const  Color errorColor = Color(0xff123456);
+
+    await tester.pumpWidget(
+      MaterialApp(
+        theme: ThemeData(colorScheme:  const ColorScheme.light(error: errorColor)),
```

Comment:
I think there are multiple spaces before the `const`.

Also nit: Maybe split the colorScheme parameter onto its own line.

Reply:
I have made it to a newline of color scheme to its own